components/Dialogs/examples/DialogsThemedPresentationExampleViewController.swift

```diff
@@ -0,0 +1,196 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialDialogs
+import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
+import MaterialComponentsBeta.MaterialDialogs_Theming
+
+class CustomDialogViewController: UIViewController {
+  var dialogTitle: String?
+  var dialogMessage: String?
+
+  init(title: String, message: String) {
+    super.init(nibName: nil, bundle: nil)
+    dialogTitle = title
+    dialogMessage = message
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.backgroundColor = UIColor.white
```

Comment:
Please use the color scheme for the example.  Especially if this is going to be "presentable", it should use the catalog theme.

Reply:
Addressed in 985551b292491379b5228a975388f1deac15e974